openmc/plotter.py

```diff
@@ -53,16 +53,18 @@
 _MAX_E = 20.e6
 
 
-def plot_xs(this, types, divisor_types=None, temperature=294., data_type=None,
-            axis=None, sab_name=None, ce_cross_sections=None,
-            mg_cross_sections=None, enrichment=None, plot_CE=True, orders=None,
-            divisor_orders=None, **kwargs):
+ELEMENT_NAMES = list(openmc.data.ELEMENT_SYMBOL.values())[1:]
+
+
+def plot_xs(this, types, divisor_types=None, temperature=294., axis=None,
+            sab_name=None, ce_cross_sections=None, mg_cross_sections=None,
+            enrichment=None, plot_CE=True, orders=None, divisor_orders=None):
     """Creates a figure of continuous-energy cross sections for this item.
 
     Parameters
     ----------
-    this : str or openmc.Material
-        Object to source data from
+    this : {str, openmc.Material}
+        Object to source data from. Nuclides and Elements can be input as a str
```

Comment:
The `{ }` notation is used to represent a set of values that are acceptable (as opposed to acceptable classes).
```suggestion
    this : str or openmc.Material
        Object to source data from. Nuclides and elements can be input as a str
```